### PR TITLE
Log cleanup

### PR DESCRIPTION
**Pull request by ClutchplateDude**
- Cleaned up logging somewhat
- Ensured all log messages have a category start word.
- Ensured all log messages are stored in Flash memory.

### CODE REVIEW

File: src/WifiControl.cpp [hunk 1]
```diff
@@ -174,31 +174,31 @@ void WifiControl::tcpLoop()
     {
         while (client.available())
         {
-            LOGV2(DEBUG_WIFI, F("WifiTCP: Available bytes %d. Peeking."), client.available());
+            LOGV2(DEBUG_WIFI, F("WIFITCP: Available bytes %d. Peeking."), client.available());
```
Comment (andre-stefanov) [inline src/WifiControl.cpp]: should not this be `WIFI: TCP:` or `[WIFI][TCP]` ?

File: src/WifiControl.cpp [hunk 2]
```diff
@@ -222,14 +222,14 @@ void WifiControl::udpLoop()
         reply += "@";
         reply += WiFi.localIP().toString();
         LOGV4(DEBUG_WIFI,
-              F("WifiUDP: Received %d bytes from %s, port %d"),
+              F("WIFIUDP: Received %d bytes from %s, port %d"),
```
Comment (andre-stefanov) [inline src/WifiControl.cpp:225]: same as with tcp

**Review by andre-stefanov — CHANGES_REQUESTED** [review]
Maybe we could place tags into [...] ?

**Comment by cameronswinoga** [comment]
Could you rename your commit message to have the same version number as the actual version? (`2.10.4` vs `1.10.4`)

**Review by julianneswinoga — APPROVED** [review]
Could you rename your commit message to have the same version number as the actual version? (`2.10.4` vs `1.10.4`)

**Pull request merged**